content/chainguard/fips/not-approved-algorithms.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "FIPS and not-approved MD5"
+linktitle: "FIPS and MD5"
+type: "article"
+description: "Technical deep-dive into Chainguard FIPS images access to not-approved algorithms such as MD5 and SHA1"
+date: 2025-10-28T08:00:00+00:00
+lastmod: 2025-10-28T08:00:00+00:00
+draft: false
+tags: ["FIPS", "MD5"]
+images: []
+weight: 040
+toc: true
+---
+
+## Overview
+
+FIPS cryptographic modules implement cryptographically strong protection of data at rest and in transit. NIST position on this is very [clear](https://csrc.nist.gov/projects/cryptographic-module-validation-program):
+
+> Non-validated cryptography is viewed as providing no protection to the information or data—in effect the data would be considered unprotected plaintext. If the agency specifies that the information or data be cryptographically protected, then FIPS 140-2 or FIPS 140-3 is applicable. In essence, if cryptography is required, then it must be validated. Should the cryptographic module be revoked, use of that module is no longer permitted.
+
+Given cryptographic strength and wide adoption and availability of NIST approved algorithms, many of them have been widely deployed for non-cryptographic purposes. For example many secure digests are often used as identifiers, one-way compression functions, hash maps, cyclic redundancy checks. They are used out of convenience, rather than their cryptographic security. In the following article allowed and unallowed usage is expalined, as well specific implementation guidance is provided for all currently implemented solutions.
+
+## NIST guidance on not-approved algorithm usage
+
+As part of FIPS collection of publications, NIST publishes [FIPS 140-3 Implementation Guidance](https://csrc.nist.gov/projects/cryptographic-module-validation-program/fips-140-3-ig-announcements) (FIPS 140 I.G.), the Cryptogrpahic Module Validation Programme (CMVP) is bound by resolutions and guidance published in FIPS 140-3 I.G. All sections inside that document are stable and are often referenced in the Security Policies and externally.
```

Comment:
Split the sentence after the IG link (so "The Cryptographic …"). The CMVP sentence should say something like "follows the IG and its referenced standards" instead of "bound by resolutions and guidance…"

Reply:
reworded and also referenced the cmvp website with pretty diagram.